README.md

```diff
@@ -84,100 +84,102 @@ Check out the repository to find examples / tests for each feature.
 Safely access to an element:
 
 ```swift
-var myArray = [1, 2, 3]
-print(myArray[safe: 0]) // Optional(1)
-print(myArray[safe: 10]) // nil
+var array = [1, 2, 3]
+print(array[safe: 0]) // Optional(1)
+print(array[safe: 10]) // nil
 ```
 
 Access to a random element:
 
 ``` swift
-var myArray = [1, 2, 3]
-print(myArray.random()) // 2 or something else
+var array = [1, 2, 3]
+print(array.random()) // 1, 2 or 3
 ```
 
-Find the indexes of an object:
+Find all the index of an object:
 
 ``` swift
-var myArray = [1, 2, 3, 1]
-print(myArray.indexes(of: 1)) // [0,3]
+var array = [1, 2, 3, 1]
+print(array.indexes(of: 1)) // [0,3]
 ```
 
-Get index of last occurrence of an object:
+Get index of first / last occurrence of an object:
 
 ``` swift
-var myArray = [1, 2, 3, 1]
-print(myArray.lastIndex(of: 1)) // 3
+var array = [1, 2, 3, 1]
+print(array.firstIndex(of: 1)) // Optional(0)
+print(array.lastIndex(of: 1)) // Optional(3)
 ```
 
 Remove an object:
 
 ``` swift
-var myArray = [1, 2, 3]
+var array = [1, 2, 3]
 myArray.remove(object: 2)
 print(myArray) // [1, 3]
 myArray.remove(objects: [1, 3])
 print(myArray) // []
 ```
 
-Check if an array contains instance of an object:
+Remove all the duplicates:
 
-``` swift
-var myArray = [1, 2, 3]
-print(myArray.contains(instanceOf: 1)) // true
-print(myArray.contains(instanceOf: "1")) // false
+```swift
+var array = [0, 0, 1, 1, 2]
+array.removeDuplicates()
+print(array) // [0,1,2]
 ```
 
-Check if an array contains another array:
+Remove all instances of an item:
 
 ``` swift
-var myArray = [1, 2, 3]
-print(myArray.contains(array: [1, 2])) // true
-print(myArray.contains(array: [5])) // false
+var array = [0, 0, 1, 1, 2]
+array.removeAll(0)
+print(array) // [1,1,2]
 ```
 
-Get an object at a specified index:
+Check if an array contains another array:
```

Comment:
I think "Check if an array contains another array" sounds like checking whether an array is a sub sequence of another array, but what the `contains` method really does is checking whether an array is a subset of another array. What I mean is `[1, 2, 3]` is a subset of `[3, 2, 1, 4, 5]` but not a sub sequence. Maybe change the description to "Check if an array is a subset of another array" or "Check if an array has all the elements in another array".